### ISSUE TITLE
manpage .Os name: time to replace "Openwall Project" with something else?

### ISSUE THREAD

**Issue by Artoria2e5** [besser82/libxcrypt#184]
libxcrypt currently says "Openwall Project" in the footer, even though (as far as I can tell) it's not really maintained or assembled together into a crypt.3 implementation by the Openwall people. https://github.com/besser82/libxcrypt/blob/72f75aa370ae96ccd2cc44ea3cf4182d8679ffbe/doc/crypt.3#L18

I do see that the man page is derived from openwall crypt_rn.3. I guess nobody just bothered to change it, not even during the mdoc conversion?

**Comment by zackw** [comment]
I preserved that on purpose during the mdoc conversion, to honor this project's origins in the Openwall Project. However, it might make sense to change it to "libxcrypt", as that would more clearly document the _current_ origin of the manpage.  @solardiz do you have an opinion?

**Comment by solardiz** [comment]
As it happens, I'm actually preparing a pull request with some edits to the man pages, prompted by the "bug" report here: https://www.openwall.com/lists/yescrypt/2024/03/21/1
I also saw this issue and also thought that I'm fine with either the mentions of `Openwall Project` staying as they are (as I had founded it and am also a bit involved here) or us switching to `libxcrypt`. Not having a clear preference, I didn't touch this in my edits so far.

**Comment by Artoria2e5** [comment]
I am thinking that a lot of the information in [AUTHORS](https://github.com/besser82/libxcrypt/blob/develop/AUTHORS) can probably go into an expanded version of `.Sh HISTORY` or a new `.Sh AUTHORS`. Exactly how many words it would be, I don't know.

**Comment by zackw** [comment]
@solardiz
> As it happens, I'm actually preparing a pull request with some edits to the man pages, prompted by the "bug" report here: https://www.openwall.com/lists/yescrypt/2024/03/21/1

I look forward to seeing it.

> I'm fine with either the mentions of Openwall Project staying as they are (as I had founded it and am also a bit involved here) or us switching to libxcrypt

How about, as part of your PR, you change all these to
```
.Os "libxcrypt (Openwall Project)"
```
which preserves the credit but also indicates the current proximate source of the manpage.

**Comment by solardiz** [comment]
> I'm actually preparing a pull request with some edits to the man pages

This is now #185. FWIW, I found the attribution to Openwall Project motivating for me to do the full proof-reading of `crypt.5`.

> How about, as part of your PR, you change all these

I sent the PR maybe a minute before you posted this comment. But I can add a commit with that change if you and @besser82 want that - let me know.

**Comment by zackw** [comment]
@Artoria2e5 I don't think that information belongs in the `crypt(3)` manpage. It might be appropriate for `crypt(5)` but I see no particular harm in leaving it where it is -- anyone who cares about these details probably wants to look at the source anyway.  What definitely _would_ make sense is to add a URL for this github repo somewhere.

**Comment by zackw** [comment]
@solardiz Thanks. I will review that ... probably on Friday, given that I have several other major time commitments at the moment.  It's up to you whether you want to add the `.Os` change; if you don't, I'll probably do it myself after giving @besser82 a week or so to comment.
